amundsen_application/static/js/components/TableDetail/ColumnListItem/index.tsx

```diff
@@ -73,6 +73,19 @@ export class ColumnListItem extends React.Component<
     e.stopPropagation();
   };
 
+  renderDescription = () => {
```

Comment:
nit: mind changing the name and return type to `shouldRenderDescription(): boolean`. This is to make it more intuitive what this does.

Reply:
yeah sure, will do